internal/cmd/run/apitest.go

```diff
@@ -15,8 +16,13 @@ import (
 	"github.com/spf13/cobra"
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
+	"golang.org/x/time/rate"
 )
 
+// Query rate is queryRequestRate per second.
+var queryRequestRate = 5
+var rateLimitTokenBucket = 20
```

Comment:
That value seems pretty high, considering that https://docs.saucelabs.com/dev/api/#rate-limit-breakdown states `10 requests per second [...]`. Though I do not know the real limits of the APIT endpoints?

Reply:
That value was tested and seems to be fine. I was not aware we have a doc for the rate limit. I'll update them according to the doc.